.github/workflows/CI-orchestrator.yml

```diff
@@ -93,27 +144,36 @@ jobs:
           fi
       - name: Prepare Slack Notification Data
         run: |
-          OVERALL_STATUS="success"
-
-          if [ "${{ needs.build-test-job.result }}" != "success" ] || 
-              [ "${{ needs.test-coverage-job.result }}" != "success" ] ||
-              [ "${{ needs.lint-format-job.result }}" != "success" ] || 
-              [ "${{ needs.e2e-test-job.result }}" != "success" ]; then
-            OVERALL_STATUS="failure"
+          if [ "${{ needs.set-overall-result.outputs.pr-url }}" != "" ]; then
+            echo "PR_LINK=<${{ needs.set-overall-result.outputs.pr-url }}|Pull Request>" >> $GITHUB_ENV
+          else
+            echo "PR_LINK=N/A" >> $GITHUB_ENV
           fi
-          
-          if [ "$OVERALL_STATUS" = "success" ]; then
+      
+          if [ "${{ needs.set-overall-result.outputs.branch-name }}" != "" ]; then
+          echo "BRANCH_LINK=<https://github.com/${{ github.event.repository.full_name }}/tree/${{ needs.set-overall-result.outputs.branch-name }}|${{ needs.set-overall-result.outputs.branch-name }}>" >> $GITHUB_ENV
```

Comment:
For coherency with the other two pieces of info (`Pull Request` and `Commit`) I would rewirte this line as:
`echo "BRANCH_LINK=<https://github.com/${{ github.event.repository.full_name }}/tree/${{ needs.set-overall-result.outputs.branch-name }}|Branch>" >> $GITHUB_ENV`

Reply:
Done